torch/_inductor/codecache.py

```diff
@@ -1597,6 +1598,12 @@ def compile(
             specified_dir=specified_output_path,
         )
 
+        # Log the AOTInductor wrapper and kernel code, if needed.
+        with tempfile.NamedTemporaryFile("w+") as t:
+            t.writelines((wrapper_code, "\n", kernel_code, "\n"))
+            t.flush()
+            V.debug.output_code(t.name, extension="cpp")
```

Comment:
I thought we are already writing out kernel.cpp and wrapper.cpp when using `TORCH_LOGS=output_code`?

Reply:
@desertfire The distinction here is that TORCH_LOGS logs to screen, which can be 5k+ lines for some models (much more than a normal terminal line buffer), where this new code piggybacks on the existing `TORCH_COMPILE_DEBUG` infrastructure to save this code to `output_code.cpp` in the `torch_compile_debug` folder. Makes it much easier to retrieve and analyze the code.